tests/functional/api/groups/members_test.py

```diff
@@ -237,6 +238,7 @@ def auth_client_header(auth_client):
 @pytest.fixture
 def group(db_session, factories):
     group = factories.Group()
+    db_session.add(GroupMembership(group_id=group.id, user_id=group.creator.id))
```

Comment:
In a future PR (https://github.com/hypothesis/h/pull/9047) I'll add new relationships that make adding memberships to groups nicer.